ArcadiaParties-SPA/angular.json

```diff
@@ -24,6 +24,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/pink-bluegrey.css",
```

Comment:
No, this should be added to `styles.css`. To be more accurate, `styles.css` should be transformed to `styles.scss` and import this theme.

Reply:
Material theme reference changed. Changed CSS to SCSS